packages/plugin-vue/tsdown.config.ts

```diff
@@ -1,14 +0,0 @@
-import { defineConfig } from 'tsdown'
-
-export default defineConfig({
-  entry: 'src/index.ts',
-  inputOptions: {
-    resolve: {
-      alias: {
-        // we can always use node version (the default entry point has browser support)
-        debug: 'debug/src/node.js',
-      },
-    },
-  },
-  dts: true,
```

Comment:
We need to keep this `dts: true`, otherwise the type definition is not generated

Reply:
Fixed now